magellan-library/src/main/java/com/wealthfront/magellan/Navigator.java

```diff
@@ -459,7 +459,7 @@ public void goBackTo(final Screen screen, NavigationType navigationType) {
     navigate(new HistoryRewriter() {
       @Override
       public void rewriteHistory(Deque<Screen> history) {
-        checkArgument(history.contains(screen), "Can't go back to a screen that isn't in history.");
+        checkArgument(history.contains(screen), String.format("Can't go back to a screen (%s) that isn't in history.", screen.getClass().getName()));
```

Comment:
do we want to include the current history as well?

Reply:
d